library/src/scripts/layout/Heading.tsx

```diff
@@ -11,7 +11,7 @@ import { typographyClasses } from "@library/styles/typographyStyles";
 export interface ICommonHeadingProps {
     id?: string;
     depth?: 1 | 2 | 3 | 4 | 5 | 6;
-    renderAsDepth?: 1 | 2 | 3 | 4 | 5 | 6;
+    renderAsDepth?: 1 | 2 | 3 | 4 | 5 | 6 | "custom";
```

Comment:
Is this component really bringing much to the table in the custom mode?

At that point we could just do

`<h4 className={customClass} />` (or whatever level you want).